src/components/views/auth/InteractiveAuthEntryComponents.js

```diff
@@ -641,7 +641,7 @@ const AuthEntryComponents = [
     TermsAuthEntry,
 ];
 
-export function getEntryComponentForLoginType(loginType) {
+export default function getEntryComponentForLoginType(loginType) {
```

Comment:
This one is a bit questionable from a code style perspective, but we haven't figured out what our rules are for default exports are so this is fine for now.

Reply:
Yeah, wasn't sure how we feel about... Easy to change again later.